src/js/common/schemaform/FieldTemplate.jsx

```diff
@@ -5,8 +5,8 @@ import React from 'react';
  */
 
 export default function FieldTemplate(props) {
-  const { id, schema, help, required, rawErrors, children, formContext, uiSchema } = props;
-  const hasErrors = (formContext.submitted || formContext.touched[id]) && rawErrors && rawErrors.length;
+  const { id, schema, help, required, rawErrors, children, formContext, touchedSchema, uiSchema } = props;
```

Comment:
I updated how we tracked touched fields in our fork of the schema library, this is to accommodate that